infra/main.bicep

```diff
@@ -32,16 +44,28 @@ var resourceToken = toLower(uniqueString(subscription().id, environmentName, loc
 var tags = { 'azd-env-name': environmentName }
 
 // Organize resources in a resource group
-resource rg 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
   name: !empty(resourceGroupName) ? resourceGroupName : '${abbrs.resourcesResourceGroups}${environmentName}'
   location: location
   tags: tags
 }
 
+resource openAiResourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' existing = if (!empty(openAiResourceGroupName)) {
```

Comment:
@briansmi - Do you think it is common for folks to have existing openai, search, or storage that aren't in the same locations?  Right now, the template is easy if all are in the same location, but more difficult if they are in different locations.  There would be more work required to support resources in different locations than the location the application is deployed to.

Is that common enough scenario that we want to make it very easy to support that?

Reply:
Created issue #33 to track